assets/base/config.json

```diff
@@ -2,5 +2,6 @@
     "DefaultServerUrl": "http://localhost:8065",
     "TestServerUrl": "http://localhost:8065",
     "DefaultLocale": "en",
-    "DefaultTheme": "default"
+    "DefaultTheme": "default",
+    "GooglePlaySenderId": "184930218130"
```

Comment:
There's nothing malicious someone could do by having this here, is there? I assume it gets compiled into the app anyway so it's not like someone couldn't get it that way if they were trying hard enough